client/web/src/nav/GlobalNavbar.tsx

```diff
@@ -376,20 +376,18 @@ export const GlobalNavbar: React.FunctionComponent<React.PropsWithChildren<Props
                             />
                         </NavAction>
                     )}
-                    {props.authenticatedUser &&
-                        (props.authenticatedUser.siteAdmin ||
-                            userExternalServicesEnabledFromTags(props.authenticatedUser.tags)) && (
-                            <NavAction>
-                                <StatusMessagesNavItem
-                                    user={{
-                                        id: props.authenticatedUser.id,
-                                        username: props.authenticatedUser.username,
-                                        isSiteAdmin: props.authenticatedUser?.siteAdmin || false,
-                                    }}
-                                    history={history}
-                                />
-                            </NavAction>
-                        )}
+                    {props.authenticatedUser?.siteAdmin && (
+                        <NavAction>
+                            <StatusMessagesNavItem
+                                user={{
+                                    id: props.authenticatedUser.id,
+                                    username: props.authenticatedUser.username,
+                                    isSiteAdmin: props.authenticatedUser?.siteAdmin || false,
```

Comment:
I don't think this can be null on this path
```suggestion
                                    isSiteAdmin: props.authenticatedUser.siteAdmin || false,
```